tests/application-gateway-legacy-tests/README.md

```diff
@@ -0,0 +1,26 @@
+# Application Gateway Tests
+
+## Overview
+
+This project contains the acceptance tests for the Kyma Application Gateway.
+
+## Prerequisites
+
+The project requires Go 1.8 or higher.
+
+## Usage
+
+Environment parameters used by the tests:
+
+| Name | Required | Default | Description | Possible values |
+|------|----------|---------|-------------|-----------------|
+| **APPLICATION** | Yes | None | The name of the Application to test | `my-application` |
+| **NAMESPACE** | Yes | None | The Namespace in which the test Application will operate | `kyma-integration` |
```

Comment:
```suggestion
| **NAMESPACE** | Yes | None | Namespace in which the test Application will operate | `kyma-integration` |
```
We tend to skip the definite article at the beginning of the description in such columns for improved readability.